File: Codeforces/A_451.java
import java.util.*;
import java.lang.*;
import java.io.*;
public class A_451{
	public static void main (String[] args) throws java.lang.Exception{
		Scanner scan = new Scanner(System.in);
		int n = scan.nextInt();
		int m = scan.nextInt();
		int mini = Math.min(n, m);
		if(mini % 2 == 0){
			System.out.println("Malvika");
		}
		else{
			System.out.println("Akshat");
		}
		scan.close();
	}
}

File: Codeforces/A_451.cpp
#include <iostream>
using namespace std;
int main()
{
    int r, c, turns = 0, nodes;
    cin >> r >> c;
    nodes = r * c;
    while (r * c != 0)
    {
        turns++;
        nodes -= (r + c - 1);
        r--;
        c--;
    }
    if (turns % 2 == 0)
        cout << "Malvika";
    else
        cout << "Akshat";
    return 0;
}